### PR TITLE
fix dark-single-file-canvas

### PR DESCRIPTION
**Pull request by OceanOak**
No changelog

This Updates the canvases to reflect the recent packages changes.

### CODE REVIEW

File: canvases/dark-repl/main.dark [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [<HttpHandler("GET", "/")>]
-let _handler _req = Http.redirectTo "/assets/index.html"
+let _handler _req =
+  PACKAGE.Darklang.Stdlib.Http.redirectTo "/assets/index.html"
```
Comment (OceanOak) [inline canvases/dark-repl/main.dark:3]: The redirect isn't working currently, I get `Header must be a string`error 
but when I try to access `http://dark-repl.dlio.localhost:11003/assets/index.html`it works fine

Comment (StachuDotNet) [reply canvases/dark-repl/main.dark:3]: Hmm this may be resolved if you rebase. I think that [this](#4923) PR would likely fix the issue.

Comment (StachuDotNet) [reply canvases/dark-repl/main.dark:3]: specifically this line: https://github.com/OceanOak/dark/compare/spike-4...darklang:dark:main#diff-2e69a7ca467f30e522c6179c7bb8f6b3eb6898577e88a197aa9fd596a9207a0eR84

(in other words: the function broke when we ported it to .dark package, and that PR fixed it)

Comment (OceanOak) [reply canvases/dark-repl/main.dark:3]: Thanks!

**Comment by StachuDotNet** [comment]
Great, thank you! Been meaning to do this :)

**Pull request merged**